src/Application/ApplicationManagement.tsx

```diff
@@ -559,18 +399,21 @@ class ManageApplicationContainer extends React.Component<
    * @param values the formik values
    * @param actions the formik actions
    */
-  private handleSubmit(values: any, actions: FormikActions<any>) {
+  private handleSubmit(values: any) {
     const { mode } = this.state;
-    let handler: Promise<boolean>;
+    let handler;
     switch (mode) {
       case ManageApplicationModes.EDIT:
-        handler = this.handleEdit(values, actions);
+        handler = this.handleEdit.bind(this);
```

Comment:
Bind `this` in the constructor so we bind only once, not every time we call the method